docs/extensions.adoc

```diff
@@ -328,7 +328,24 @@ class TimedSpec extends Specification {
 When applied to a fixture method, the timeout is per execution of the fixture method.
 
 When a timeout is reported to the user, the stack trace shown reflects the execution stack of the test framework when
-the timeout was exceeded.
+the timeout was exceeded. Additionally, thread dumps are captured and logged on unsuccessful interrupt attempts.
+
+If you want to customize the thread dump capturing behavior, you can use the <<spock-configuration-file,Spock Configuration File>>.
+
+.Timeout Configuration
+[source,groovy]
+----
+timeout {
+  // boolean, default true
```

Comment:
💬 I feel like it would be better to communicate what these properties do, but that won't align with how we document other configuration objects.